environments/dev/manifests/loki-config-map.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: v1
+data:
+  config.yaml: |-
+    "auth_enabled": true
+    "chunk_store_config":
+      "chunk_cache_config":
+        "memcached":
+          "batch_size": 100
+          "parallelism": 100
+        "memcached_client":
+          "addresses": "dns+observatorium-xyz-loki-chunk-cache.observatorium.svc.cluster.local:11211"
+          "consistent_hash": false
+          "max_idle_conns": 100
+          "timeout": "100ms"
+          "update_interval": "1m"
+      "max_look_back_period": "2h"
+      "write_dedupe_cache_config":
+        "memcached":
+          "batch_size": 100
+          "parallelism": 100
+        "memcached_client":
+          "addresses": "dns+observatorium-xyz-loki-index-write-cache.observatorium.svc.cluster.local:11211"
+          "consistent_hash": true
+    "distributor":
+      "ring":
+        "kvstore":
+          "etcd":
+            "endpoints":
+            - "observatorium-xyz-loki-ring-store.observatorium.svc.cluster.local:2379"
+          "store": "etcd"
+    "frontend":
+      "compress_responses": true
+      "max_outstanding_per_tenant": 200
+    "frontend_worker":
+      "frontend_address": "observatorium-xyz-loki-query-frontend.observatorium.svc.cluster.local:9095"
+      "grpc_client_config":
+        "max_send_msg_size": 104857600
+      "parallelism": 8
+    "ingester":
+      "chunk_block_size": 262144
+      "chunk_idle_period": "15m"
+      "lifecycler":
+        "heartbeat_period": "5s"
+        "interface_names":
+        - "eth0"
+        "join_after": "30s"
+        "num_tokens": 512
+        "ring":
+          "heartbeat_timeout": "1m"
+          "kvstore":
+            "etcd":
+              "endpoints":
+              - "observatorium-xyz-loki-ring-store.observatorium.svc.cluster.local:2379"
+            "store": "etcd"
+          "replication_factor": 1
+      "max_transfer_retries": 60
+    "ingester_client":
+      "grpc_client_config":
+        "max_recv_msg_size": 67108864
+      "remote_timeout": "1s"
+    "limits_config":
+      "enforce_metric_name": false
+      "ingestion_burst_size_mb": 32
+      "ingestion_rate_mb": 32
+      "ingestion_rate_strategy": "global"
+      "max_global_streams_per_user": 10000
+      "max_query_length": "12000h"
+      "max_query_parallelism": 32
+      "max_streams_per_user": 0
+      "reject_old_samples": true
+      "reject_old_samples_max_age": "168h"
+    "querier":
+      "engine":
+        "max_look_back_period": "2h"
```

Comment:
The default value for this is better 3m. Basically for instant query how far back we're looking for data.

Reply:
Actually scratch that you might want to look at 2h for instant query. But that's what this is for. Doesn't really impact all other queries.